ratatui-crossterm/src/lib.rs

```diff
@@ -6,16 +6,40 @@
     html_favicon_url = "https://raw.githubusercontent.com/ratatui/ratatui/main/assets/favicon.ico"
 )]
 #![warn(missing_docs)]
-//! This module provides the [`CrosstermBackend`] implementation for the [`Backend`] trait. It uses
-//! the [Crossterm] crate to interact with the terminal.
+//! This crate provides [`CrosstermBackend`], an implementation of the [`Backend`] trait for the
+//! [Ratatui] library. It uses the [Crossterm] library for all terminal manipulation.
 //!
+//! ## Crossterm Version and Re-export
+//!
+//! `ratatui-crossterm` requires you to specify a version of the [Crossterm] library to be used.
+//! This is managed via feature flags. You **must** enable one, and only one, of the available
+//! `crossterm_xx` features (e.g., `crossterm_28`, `crossterm_29`). These features determine which
+//! version of Crossterm is compiled and used by the backend. Attempting to enable none or multiple
```

Comment:
```suggestion
//! version of [Crossterm] is compiled and used by the backend. Attempting to enable none or multiple
```

Reply:
We don't need to link every time this is mentioned. It's enough to link it in the first paragraph I think. Sound good?

Reply:
I'd link every time due to OCD... but sure :D